src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -251,7 +251,7 @@ if ($saveConfig) {
         throw "Cannot edit config file in dev mode, please add $user to ""$iisAdminOwners"" group manually"
     }
     ## Check the current user is a member of the Administrators local user group
-    if (!(Get-LocalGroupMember -SID $administratorsSID -Member $user -ErrorAction SilentlyContinue)) {
+    if (!(Get-LocalGroupMember -SID $administratorsSID -ErrorAction SilentlyContinue | Where-Object { $_.Name -eq $user } )) {

```

Comment:
are you sure -eq works? I thought only -like would work

Reply:
Also, since it is a non-trivial workaround, a comment is required

Reply:
yes, the -eq works.
I added some comments.